frontend/src/app/main/ui/workspace/sidebar/options/menus/interactions.cljs

```diff
@@ -184,6 +184,8 @@
         destination          (get objects (:destination interaction))
 
         frames               (mf/with-memo [objects] (ctt/get-viewer-frames objects {:all-frames? true}))
+        shape-parent-ids     (mf/with-memo [objects] (cph/get-parent-ids objects (:id shape)))
+        shape-parents        (filter (comp (set shape-parent-ids) :id) frames)
```

Comment:
Think this one also needs to be wrapped in `(mf/with-memo [frames]`, to avoid repeating the filter each time the component is repainted.

Reply:
Ok!